frontend/storybook/src/components/entry/publish/pages/publish-two.ts

```diff
@@ -1,86 +1,71 @@
-// import "@elements/entry/jig/publish/page-2";
-// import "@elements/entry/home/TOSORT/image-thumbnail"; 
-// import "@elements/entry/home/TOSORT/placeholder"; 
-// import "@elements/widgets/tags/icon";
-// import "@elements/entry/jig/publish/publish-dropdown";
-// import "@elements/core/inputs/textarea";
-// import "@elements/core/dividers/spacer-fourty";
-// import "@elements/entry/jig/publish/dropdown-list-object";
-// import "@elements/entry/jig/publish/publish-embed";
-// import { Rectangle } from "~/components/core/buttons/rectangle";
-// import {Color, Size, IconAfter, IconBefore} from "@elements/core/buttons/rectangle";
-// import { colorStyles } from "@elements/_styles/colors";
-// import "@elements/widgets/tooltips/right";
-// export default {
-//   title: 'Entry/Jig/Publish/Pages',
-// }
+import "@elements/entry/jig/publish/page-2";
+import "@elements/entry/home/TOSORT/image-thumbnail"; 
+import "@elements/entry/home/TOSORT/placeholder"; 
+import "@elements/widgets/tags/icon";
+import "@elements/entry/jig/publish/publish-dropdown";
+import "@elements/core/inputs/textarea";
+import "@elements/core/dividers/spacer-fourty";
+import "@elements/entry/jig/publish/dropdown-list-object";
+import "@elements/entry/jig/publish/publish-embed";
+import { Rectangle } from "~/components/core/buttons/rectangle";
+import "@elements/widgets/tooltips/right";
+export default {
+  title: 'Entry/Jig/Publish/Pages',
+}
 
   
-//   interface PublishArgs {
-//     closed:boolean,
-//     embedclosed:boolean,
+  interface PublishArgs {
+    closed:boolean,
+    embedclosed:boolean,
   
-//     }
+    }
 
-//     const DEFAULT_ARGS:PublishArgs = {
-//        closed:false,
-//        embedclosed:false,
-//       }
-
-      
-//       const STR_MEDIUM = "medium";
-//       const STR_WHITE = "white";
-//       const STR_SHARE = "share the JIG";
-//       const STR_CREATE = "create a new JIG";
-//       const STR_PLAY = "play the JIG";
-//       const STR_ICNSHARE = "Icn_Share_Red.svg";
-//       const STR_ICNCREATE = "Icn_Plus_Red.svg";
-//       const STR_ICNPLAY = "icn-video-activity-hover.svg";
-//       const STR_DROPDOWNTITLE = "Select Share Option";
-//       const STR_STUDENT = "icn-student.svg";
-//       const STR_SHARESTUDENT ="Share with Students";
-//       const STR_URLICON = "icn-url.svg";
-//       const STR_URL ="Copy URL";
-//       const STR_EMBED ="Embed this JIG";
-//       const STR_EMBEDICON ="Icn_Embed.svg";
-
-      
+    const DEFAULT_ARGS:PublishArgs = {
+       closed:false,
+       embedclosed:false,
+      }
 
+      const STR_SHARE = "share the JIG";
```

Comment:
Shouldn't these be in an element?

Reply:
Its the string for a rectangle button so I believe its best to be kept here